net-p2p/qbittorrent/qbittorrent-4.1.2-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils xdg-utils
+
+DESCRIPTION="BitTorrent client in C++ and Qt"
+HOMEPAGE="https://www.qbittorrent.org
+	  https://github.com/qbittorrent"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/${PN}/qBittorrent.git"
+else
+	SRC_URI="https://github.com/qbittorrent/qBittorrent/archive/release-${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~ppc64 ~x86"
+	S="${WORKDIR}/qBittorrent-release-${PV}"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="+dbus debug webui +X"
+REQUIRED_USE="dbus? ( X )"
+
+RDEPEND="
+	>=dev-libs/boost-1.62.0-r1:=
+	dev-qt/qtcore:5
+	dev-qt/qtnetwork:5[ssl]
+	>=dev-qt/qtsingleapplication-2.6.1_p20130904-r1[qt5(+),X?]
+	dev-qt/qtxml:5
+	>=net-libs/libtorrent-rasterbar-1.0.6:0=
+	sys-libs/zlib
+	dbus? ( dev-qt/qtdbus:5 )
+	X? (
+		dev-libs/geoip
+		dev-qt/qtgui:5
+		dev-qt/qtsvg:5
+		dev-qt/qtwidgets:5
+	)"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig"
+
+DOCS=( AUTHORS Changelog CONTRIBUTING.md README.md TODO )
+
+src_configure() {
+	econf --with-qtsingleapplication=system \
+	$(use_enable dbus qt-dbus) \
+	$(use_enable debug) \
+	$(use_enable webui) \
+	$(use_enable X gui)
+}
+
+src_install() {
+	emake STRIP="/bin/false" INSTALL_ROOT="${D}" install
+	einstalldocs
+	insinto /usr/share/applications
```

Comment:
1.) please do it steaight after emake install
2.) please do not do it "by hands"
consult here: https://devmanual.gentoo.org/eclass-reference/desktop.eclass/index.html

Reply:
Also do not believe it worth a revision bump

Reply:
Fixed.

(Although now people who upgraded already will not get the desktop file.)

Reply:
As I figured out it is the common problem (I have no idea why my whisker menu showed the icon after 4.1.2 upgrade), so they will try to update it in any way

Reply:
You probably had some form of local override of the .desktop file? Like in ~/.local/share/applications. I do have a couple of overrides myself for badly written .desktop files.

In any event, this:

    qlist qbittorrent

shows that no desktop file was being installed by the package.